presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/CommandExecutor.java

```diff
@@ -13,23 +13,42 @@
  */
 package com.teradata.prestomanager.agent;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
 
 import static java.lang.Runtime.getRuntime;
 import static java.lang.String.format;
 
-// TODO: Add logger
 public final class CommandExecutor
 {
+    private static final Logger LOGGER = LogManager.getLogger(CommandExecutor.class);
+
     private CommandExecutor() {}
 
     public static int executeCommand(String command)
             throws PrestoManagerException
     {
+        LOGGER.debug("Command to be executed: {}", command);
+
         try {
             Process process = getRuntime().exec(command);
+            String output = new BufferedReader(new InputStreamReader(process.getInputStream()))
+                    .lines().collect(Collectors.joining(System.getProperty("line.separator")));
+            String error = new BufferedReader(new InputStreamReader(process.getErrorStream()))
+                    .lines().collect(Collectors.joining(System.getProperty("line.separator")));
+            LOGGER.info("Standard output from command : {} \n{}", command, output);
+            if (!"".equals(error)) {
+                LOGGER.warn("Standard error from command : {} \n{}", command, error);
+            }
             process.waitFor(2, TimeUnit.MINUTES);
```

Comment:
I don't think this closes the process if it hasn't exited, it just waits around for that amount of time or until the process exits, whichever comes first (haven't tried it out, just based on the docs).  You may want to force terminate the process.  in any case process.exitValue() will throw an error if the process hasn't exited.